settings.gradle

```diff
@@ -53,5 +56,13 @@ project(':detector-resources').projectDir =
 project(':e2e-test-server').projectDir =
 		"$rootDir/e2e-test-server" as File
 
-project(':examples-spring').projectDir =
-		"$rootDir/examples/spring" as File
+// Unable to resolve milestone artifact currently.
```

Comment:
Please update to the latest main branch -- this sub-project has been re-enabled.

Reply:
I did, must be a merge conflict.